### PR TITLE
Fixes #7728: Document how to switch to UDP for reporting when upgrading to >= 3.1

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7728

### CODE REVIEW

File: 6_handbook/20_application_tuning.txt [hunk 1]
```diff
@@ -204,15 +204,25 @@ mount /var/rudder/cfengine-community/state
 
 ==== Rsyslog
 
-On Rudder policy servers (root or relay), when managing a large number of nodes,
-you can experience issues with rsyslog. This happens because Rudder uses TCP by
-default for sending reports to rsyslog, which implies the system has to keep track of
+If you upgraded your Rudder server from a pre-3.1 version or if you have changed the
```
Comment (jooooooon) [inline 6_handbook/20_application_tuning.txt]: This turn of phrase is odd. How about "If you're using Rudder 3.1 or later"... ?

File: 12_upgrade/05_caution.txt [hunk 2]
```diff
@@ -13,6 +13,19 @@ However, Rudder 3.1 provides a new inventory signing feature which will only be
 These agents are not compatible with Rudder 3.1, and you have to upgrade them. 
 Be careful to follow the upgrade path explained <<_upgrade_from_rudder_2_10_or_older, above>>.
 
+==== Protocol for reporting
+
+Rudder 3.1 uses UDP by default for reporting, but if you upgrade your server from
```
Comment (jooooooon) [inline 12_upgrade/05_caution.txt]: We should mention here that the reporting protocol is in fact syslog.

**Comment by amousset** [comment]
PR updated

File: 6_handbook/20_application_tuning.txt (hunk 1 above)
Comment (jooooooon) [inline 6_handbook/20_application_tuning.txt]: Ahhhh, I understand this now I've re-read the section from 12_upgrade/05_caution.txt!

This needs explaining again here, maybe like this:

> Rudder 3.1 and later use UDP by default for reporting, but if you upgraded your server from a previous version, you will keep the previous setting which is TCP.

File: 6_handbook/20_application_tuning.txt [hunk 3]
```diff
@@ -204,15 +204,25 @@ mount /var/rudder/cfengine-community/state
 
 ==== Rsyslog
 
-On Rudder policy servers (root or relay), when managing a large number of nodes,
-you can experience issues with rsyslog. This happens because Rudder uses TCP by
-default for sending reports to rsyslog, which implies the system has to keep track of
+If you are using syslog over TCP as reporting protocol (it is set in *Administration* -> *Settings* -> *Protocol*), 
+you can experience issues with rsyslog on Rudder
+policy servers (root or relay) when managing a large number of nodes.
+This happens because using TCP implies the system has to keep track of
```
Comment (jooooooon) [inline 6_handbook/20_application_tuning.txt]: Yeah, this is great :)

File: 12_upgrade/05_caution.txt [hunk 4]
```diff
@@ -13,6 +13,20 @@ However, Rudder 3.1 provides a new inventory signing feature which will only be
 These agents are not compatible with Rudder 3.1, and you have to upgrade them. 
 Be careful to follow the upgrade path explained <<_upgrade_from_rudder_2_10_or_older, above>>.
 
+==== Protocol for reporting
+
+Rudder 3.1 uses syslog messages over UDP by default for reporting, but if you upgrade
```
Comment (jooooooon) [inline 12_upgrade/05_caution.txt]: upgradeD

File: 12_upgrade/05_caution.txt [hunk 5]
```diff
@@ -13,6 +13,20 @@ However, Rudder 3.1 provides a new inventory signing feature which will only be
...
+your server from a previous version, you will keep the previous setting which is
+are syslog messages over TCP.
```
Comment (jooooooon) [inline 12_upgrade/05_caution.txt]: s/is are/uses/

**Comment by amousset** [comment]
PR updated

**Pull request merged**